build_run_docker.sh

```diff
@@ -278,11 +257,10 @@ RONDB_VERSION_NO_DOT=$(echo "$RONDB_VERSION" | tr -d '.')
 FILE_SUFFIX="v${RONDB_VERSION_NO_DOT}_m${NUM_MGM_NODES}_g${NODE_GROUPS}_r${REPLICATION_FACTOR}_my${NUM_MYSQL_NODES}_api${NUM_API_NODES}"
 
 AUTOGENERATED_FILES_DIR="$SCRIPT_DIR/autogenerated_files/$FILE_SUFFIX"
-rm -rf $AUTOGENERATED_FILES_DIR
```

Comment:
Why not just delete it all? Are you expecting to restart a cluster with previous data?

Reply:
This came along by mistake when I reverted some of your changes. I'll put it back in.